docs/vectorizer/document-vectorization.md

```diff
@@ -0,0 +1,525 @@
+# Document Vectorization in PGAI
```

Comment:
Shall we mention the tutorial somewhere? https://github.com/timescale/pgai/tree/main/examples/embeddings_from_documents